server/protocols/webadmin/webadmin-data/src/main/java/org/apache/james/webadmin/routes/RegexMappingRoutes.java

```diff
@@ -87,9 +87,10 @@ public void define(Service service) {
     public HaltException addRegexMapping(Request request, Response response) throws Exception {
         RegexMappingDTO regexMappingDTO = jsonExtractor.parse(request.body());
         MappingSource mappingSource = extractMappingSource(regexMappingDTO);
+        String regex = extractRegex(regexMappingDTO);
 
         try {
-            recipientRewriteTable.addRegexMapping(mappingSource, extractRegex(regexMappingDTO));
+            recipientRewriteTable.addRegexMapping(mappingSource,regex);
```

Comment:
miss a space now... checkstyle :)

Reply:
I checked and fixed. I will be more careful about checkstyles and indents.